app/controllers/roadmap/course-extension-ideas.ts

```diff
@@ -29,6 +29,12 @@ export default class CourseExtensionIdeasController extends Controller {
     return this.model.courseExtensionIdeas
       .mapBy('course')
       .uniq()
+      .filter((course) => {
+        // Check if this course has any extension ideas with status "not_started" or "in_progress"
+        const courseExtensionIdeas = this.model.courseExtensionIdeas.filterBy('course', course);
```

Comment:
This is okay, but a simpler way to do this - add `rejectBy('releaseStatusIsReleased')` to the first line before `mapBy('course')`. In general prefer the shorthand methods unless they compromise readability, in general they're easier to quickly read through.